plugins/lookup/dig.py

```diff
@@ -95,6 +96,21 @@
     msg: "MX record for gmail.com {{ item }}"
   with_items: "{{ lookup('community.general.dig', 'gmail.com./MX', wantlist=true) }}"
 
+- name: lookup multiple names at once
```

Comment:
```suggestion
- name: Lookup multiple names at once
```
(All examples should be like this, but ... let's start fixing new ones ;) )

Reply:
Done.